docs/_tutorials/getting_started.md

```diff
@@ -188,7 +188,7 @@ This is a basic example, but it gives you a place to start customizing your app
 
 ### Sending and responding to actions
 
-To use features like buttons, select menus, datepickers, dialogs, and message actions, you’ll need to enable interactivity. Similar to events, you'll need to specify a URL for Slack to send the action (such as *user clicked a button*).
+To use features like buttons, select menus, datepickers, and dialogs, you’ll need to enable interactivity. Similar to events, you'll need to specify a URL for Slack to send the action (such as *user clicked a button*).
```

Comment:
@seratch this change might also be needed in Japanese docs

Reply:
Thanks - updated in the PR to your branch.